Updates Metamask Page with new steps

albertov19 opened the pull request:
### Description

Adds new Metamask flow and revamps the page a bit

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.

### After Translation Requirements

- [x] Will need to create PR in `moonbeam-docs` repo to remove images

#### Items to be Updated

Delete all of the metamask/metamask-* images and replace with the ones in the metamask/new/ directory

eshaben commented on tokens/connect/metamask.md:
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
+![Add network in Metamask](/images/tokens/connect/metamask/new/metamask-8.png)
 
-![Initiating a token transfer](/images/tokens/connect/metamask/metamask-8.png)
+Once you've added the network, you'll be redirected to a screen stating that you've successfully added a network. Furthermore, you'll be prompted a button to **Switch to Moonbase Alpha**, the network added in this example.
```
```suggestion
Once you've added the network, you'll be redirected to a screen stating that you've successfully added a network. Furthermore, you'll be prompted to **Switch to Moonbase Alpha**, the network added in this example.
```

eshaben commented on tokens/connect/metamask.md:
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
+## Interact with the Network {: #interact-with-network } 
 
-Note that the Account 2 balance has been decreased by the sent amount + gas fees. Flipping over to Account 1, you can see the 100 sent tokens have arrived:
+Once you've [connected Metamask](#connect-metamask-to-moonbeam) to any Moonbeam-based Network, you can start using your wallet by:
```
```suggestion
Once you've [connected Metamask](#connect-metamask-to-moonbeam) to any Moonbeam-based network, you can start using your wallet by:
```

eshaben commented on tokens/connect/metamask.md:
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
+
+1. Ensure you are connected to the correct network
+2. Ensure you have selected the account you want to use for the transfer
+3. In the main screen of your Metamask wallet, click on **Send**
```
```suggestion
3. On the main screen of your Metamask wallet, click on **Send**
```

eshaben commented on tokens/connect/metamask.md:
```diff
@@ -109,31 +121,62 @@ Here you can configure MetaMask for the following networks:
+
+![Select account to send tokens to in Metamask](/images/tokens/connect/metamask/new/metamask-11.png)
+
+In the next screen, take the following steps:
```
```suggestion
On the next screen, take the following steps:
```

eshaben reviewed (commented):
Thanks for updating this page! It needed some love!

Mostly tiny things, but I did notice that some of the images (some are worse than others) have white around the edges, which kind of stands out on the dark background. There are also a good number of images with a grey line that goes like half way across the bottom of the image.

I could probably update these tomorrow (Friday), if you don't get to it first

albertov19 commented:
@eshaben thanks for the thorough review.

The weird lines in the images is due to the screenshot process in my laptop, got to look into that because it is very annoying haha. 

I've addressed all the feedback

eshaben reviewed (approved):
LGTM ✨
